stable/gemini/templates/deployment.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "gemini.fullname" . }}-controller
+  labels:
+    app: gemini
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gemini
+  template:
+    metadata:
+      labels:
+        app: gemini
+    spec:
+      serviceAccountName: {{ include "gemini.fullname" . }}-controller
```

Comment:
I guarantee someone is going to request making this configurable. Just a heads up. Goldilocks has a decent example of how to make a default if not configured.